DEVELOPMENT.md

```diff
@@ -92,37 +86,8 @@ npm run demos
 
 which generates all examples and starts the demos app.
 
-## Custom TSLint rules
-
-Run the following command to build the custom TSLint rules
-
-```
-npm run tslint:build
-```
-
-To test the rules run
-
-```
-npm run tslint:test
-```
-
-### Using Gradle build
-
-Gradle build is meant for CI servers and does not require NodeJS installed
-upfront. Instead, it downloads NodeJS binaries locally from Arifactory and runs
-any yarn task with that node version.
-
-Gradle tasks look very similar to the NPM ones, e.g.:
-
-```
-./gradlew yarn_install
-./gradlew test
-./gradlew lint
-./gradlew compile
-```
-
-To see complete list of gradlew builds, run:
+If you want to run the design system itself including the demos embedded use
 
 ```
-./gradlew tasks
+npm run barista
```

Comment:
barista task does not exist in the package.json scripts

Reply:
it does ;) https://github.com/dynatrace-oss/barista/blob/master/package.json#L33